cmd/av/stack_next.go

```diff
@@ -39,6 +30,39 @@ child branch (without recursively syncing further descendants).
 			return errors.New("invalid number (must be >= 1)")
 		}
 
-		return errors.New("unimplemented")
+		// Get the subsequent branches so we can checkout the nth one
+		repo, _, err := getRepoInfo()
+		if err != nil {
+			return err
+		}
+		branches, err := meta.ReadAllBranches(repo)
+		if err != nil {
+			return err
+		}
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+		subsequentBranches, err := subsequentBranches(branches, currentBranch)
+		if err != nil {
+			return err
+		}
+
+		// confirm we can in fact do the operation given current branch state
+		if subsequentBranches == nil {
```

Comment:
Prefer `len(subsequentBranches) == 0` here (the `len` of a nil slice in Go is zero).